### PR TITLE
Add a note alerting developers

### PR DESCRIPTION
**Pull request by h4ckninja**
Returning a raw response will introduce XSS when user input is sent back. Thoughts? Just taking a stab at this. I think these screencasts are a great place to introduce some security concepts to developers and how and where developers should be aware of potential issues.

### CODE REVIEW

File: knpu/ep1-intro/routing-wildcards.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ The important thing is that the routing wildcard matches the variable name.
 
 To test this is, change the message in the response to `'The genus: '.$genusName`:
 
+**NOTE:** Returning a raw response should only be done in development as it will introduce a security issue called XSS. You can [read more about XSS here.](https://www.owasp.org/index.php/Cross-site_Scripting_%28XSS%29)
```
Comment (bocharsky-bw) [inline knpu/ep1-intro/routing-wildcards.md]: Better to use our internal **TIP** block:

```
***TIP
// your tip here...
***
```

You can get example from [downloading-the-project](https://github.com/knpuniversity/symfony/blob/master/knpu/ep1-intro/start-project.md#downloading-the-project)  chapter.

**Comment by bocharsky-bw** [comment]
:+1: to have this note.

But I think we should add it after `[[[ code('086c4c19c1') ]]]` code block.

**Comment by h4ckninja** [comment]
Can do. :) I'll commit your recommendations.

**Comment by h4ckninja** [comment]
Done.

File: knpu/ep1-intro/routing-wildcards.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ To test this is, change the message in the response to `'The genus: '.$genusName
 
 [[[ code('086c4c19c1') ]]]
 
+***TIP
+Returning a raw response should only be done in development as it will introduce a security issue called XSS. You can [read more about XSS here.](https://www.owasp.org/index.php/Cross-site_Scripting_%28XSS%29)
```
Comment (bocharsky-bw) [inline knpu/ep1-intro/routing-wildcards.md:23]: I prefer to use link to the wikipedia here:
https://en.wikipedia.org/wiki/Cross-site_scripting

Comment (h4ckninja) [inline knpu/ep1-intro/routing-wildcards.md:23]: OWASP is considered the authoritative body on web application security. What information would you like to see presented to visitors from Wikipedia that's not on OWASP?

**Comment by weaverryan** [comment]
Thanks @micheal :). I'm happy also with the OWASP link, it's one of the rare cases where it has even more authority than Wikipedia.

**Comment by h4ckninja** [comment]
:+1:

**Pull request merged**